frontend/src/components/common/header/index.css

```diff
@@ -90,20 +90,20 @@ a.dropdown-item {
 
 /* donate button on second navbar (green) */
 #donate-button {
+  margin-left: 5 0px;
```

Comment:
Good call on going with changing the `margin-left` here but sadly this is not valid CSS. I believe 5 needs to be 5px and also a `margin-left` can not have 2 values with it. 

When we see something like `margin: 5px 0px` -> the 5px refers to the `margin-top` & `margin-bottom` while the 0px refers to the `margin-left` and `margin-right` 

I think for this story we will want something like `margin-left: 20px` but that will not work at each breakpoint, so we will need to vary it a bit using media queries.